### PR TITLE
[easy] Update specs to include rotation

### PR DESCRIPTION
**Pull request by querolita**
<s>I have traced down the katex specs error that has been showing recently every time the `mdbook serve` is being executed. The first time this error appears is on November 27, commit `d1858a61c4da8ada74b4d8a9de387c6a10b96dc5` named [Fix the mdbook build issue coming from CI(gh-page.yml)](https://github.com/o1-labs/proof-systems/commit/d1858a61c4da8ada74b4d8a9de387c6a10b96dc5). This was a commit that simply added the `[output.katex]` line to the `book.toml`, as suggested by Clippy. This PR tries to revert the error, and includes missing gates in the kimchi specs that were not correctly referenced.<s>

This PR includes missing gates in the kimchi specs that were not correctly referenced, and fixes some styling issues.

### CODE REVIEW

File: book/specifications/kimchi/template.md [hunk 1]
```diff
@@ -269,17 +269,23 @@ TODO: for each gate describe how to create it?
...
 #### Xor
 
 {sections.xor16}
 
-#### Not
+##### Not
 
 {sections.not_gadget}
+
 ##### And
```
Comment (mimoo) [inline book/specifications/kimchi/template.md:281]: looks H4 instead of H5 is needed here as well

File: book/specifications/kimchi/template.md [hunk 2]
```diff
@@ -269,17 +269,23 @@ TODO: for each gate describe how to create it?
 #### Foreign Field Multiplication
 
 {sections.foreign_field_mul}
+
 #### Xor
 
 {sections.xor16}
 
-#### Not
+##### Not
```
Comment (mimoo) [inline book/specifications/kimchi/template.md:277]: if XOR is an h4, why is this an h5 :o?

File: book/specifications/kimchi/template.md (hunk 1 above)
Comment (querolita) [reply book/specifications/kimchi/template.md:281]: Oh, I intentionally wrote a fifth level as an attempt to show that none of And or Not are actual gate types, but rather gadgets reusing Xor in some manner. But I can move it to H4 if it makes more sense.

File: book/book.toml [hunk 3]
```diff
@@ -14,7 +14,6 @@ additional-css = ["./assets/css/mdbook-admonish.css", "././mdbook-admonish.css"]
 additional-js = ["mermaid.min.js", "mermaid-init.js"]
 
 # for LaTeX
-[output.katex]
```
[the remark this answers is not in the document]
Comment (querolita) [reply book/book.toml:17]: I will try that thanks!

File: book/specifications/kimchi/template.md (hunk 1 above)
Comment (mimoo) [reply book/specifications/kimchi/template.md]: ah no, that makes sense

Comment (mimoo) [reply book/specifications/kimchi/template.md]: but maybe they should be moved to a gadget section then?

**Review by mimoo — APPROVED** [review]
nice!

**Pull request closed without merging**